test/decl/var/lazy_properties.swift

```diff
@@ -41,9 +41,9 @@ class TestClass {
 
   lazy var k : Int = { () -> Int in return 0 }()+1  // multi-stmt closure
 
-  lazy var l : Int = 42 {  // expected-error {{lazy properties must not have observers}} {{3-8=}}
-    didSet {
-    }
+  lazy var l : Int = 42 {  // Okay
+    didSet {}
+    willSet {}
```

Comment:
Is it worth testing the case where only one of the two observers is defined (and in SILGen also)?